august docs changle log entry (#1499)
temporalio/documentation: change-log/august-release-2022.md

---
slug: august-release-2022
title: August 23 2022
tags:
  - docs-change-log
date: 2022-07-25T00:00:00Z
---

> - We have added a [Getting started with Temporal Cloud guide](/cloud/how-to-get-started-with-temporal-cloud)!

Reviewer: ```suggestion
- We added a [Getting started with Temporal Cloud](/cloud/how-to-get-started-with-temporal-cloud) guide!
```

> - There are also guides to [managing certificates](/cloud/how-to-manage-certificates-in-temporal-cloud) and [managing Namespaces](/cloud/how-to-manage-namespaces-in-temporal-cloud) in Temporal Cloud.
> - Blog has moved from docs.temporal.io to [temporal.io/blog](https://temporal.io/blog).

Reviewer: ```suggestion
- We also added guides for [managing certificates](/cloud/how-to-manage-certificates-in-temporal-cloud) and [managing Namespaces](/cloud/how-to-manage-namespaces-in-temporal-cloud) in Temporal Cloud.
- The Temporal blog moved from docs.temporal.io to [temporal.io/blog](https://temporal.io/blog).
```

> - A new [KB articles](https://docs.temporal.io/kb) section that introduces a “general knowledge base article” information archetype to Temporal’s Platform documentation.
>   This information archetype aims to answer questions such as “what is an aggregation of things of this theme?”, “what are similarities and differences between things”, “how to do something that bridges Temporal and other tech stacks”, “how to do something that is less common”, etc…

Reviewer: ```suggestion
- A new [KB articles](https://docs.temporal.io/kb) section introduces a “general knowledge base article” information archetype to the Temporal Platform documentation.
  This information archetype aims to answer questions such as “what is an aggregation of things of this theme?”, “what are similarities and differences between things?”, “how to do something that bridges Temporal and other tech stacks”, “how to do something that is less common”, and so on.
```
